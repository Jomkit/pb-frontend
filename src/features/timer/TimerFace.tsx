const TimerFace = ({sec, min}: {sec: number, min: number}) => {
    let time = sec < 10 ? `${min}:0${sec}` : `${min}:${sec}`;
  return (
    <>
        <div className='text-center'>
            <h3 className='text-2xl bg-slate-200 w-fit m-auto px-4 py-1 rounded border-2 border-slate-500'>{time}</h3>
        </div>
        

    </>
  )
}

export default TimerFace