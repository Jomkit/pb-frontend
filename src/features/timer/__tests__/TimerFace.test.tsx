import { render } from "@testing-library/react"
import TimerFace from "../TimerFace"

it("should render without crashing", () => {
    render(<TimerFace min={10} sec={15} />)
})

it("should match snapshot", () => {
    const { asFragment } = render(<TimerFace min={10} sec={15} />)
    expect(asFragment()).toMatchSnapshot();
})