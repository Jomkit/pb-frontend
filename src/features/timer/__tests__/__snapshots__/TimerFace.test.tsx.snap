// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="text-center"
  >
    <h3
      class="text-2xl bg-slate-200 w-fit m-auto px-4 py-1 rounded border-2 border-slate-500"
    >
      10:15
    </h3>
  </div>
</DocumentFragment>
`;
